Sources/SwifterSwift/UIKit/UIViewExtensions.swift

```diff
@@ -329,6 +329,20 @@ public extension UIView {
         return UINib(nibName: name, bundle: bundle).instantiate(withOwner: nil, options: nil)[0] as? UIView
     }
 
+    /// SwifterSwift: Load view of a certain type from nib
+    ///
+    /// - Parameters:
+    ///   - withClass: UIView type.
+    ///   - bundle: bundle of nib (default is nil).
+    /// - Returns: UIView
+    class func loadFromNib<T: UIView>(withClass name: T.Type, bundle: Bundle? = nil) -> T {
```

Comment:
Since we are still here, one question what do you think about the naming `instanceFromNib`? It is just curiosity, I found the name `load` a bit misleading since the action is really to instantiate ... but is more a question than an actual comment :)

Reply:
The name was vaguely chosen to match `loadNibNamed`. Don't have a strong opinion, though.

Reply:
I like the current name as it basically replaces the one above. It's also what I call it in my project except that I have something a little sexier:
```Swift
public extension UIView {
    class func loadFromNib(in bundle: Bundle = .main) -> Self? {
        return bundle.loadFirstObject(inNibNamed: String(describing: self))
    }
}
```
The only reason I can do it like that is because I have another extension:
```Swift
public extension Bundle {
    func loadFirstObject<T>(inNibNamed name: String, owner: Any? = nil, options: [UINib.OptionsKey: Any]? = nil) -> T? {
        return loadNibNamed(name, owner: owner, options: options)?.first as? T
    }
}
```

Reply:
That's nice too. No objection if you want to scrap this and add UIView.loadFromNib -> Self

Reply:
We're limited in this project to never call other functions because it's designed to be copyable piecemeal. It's a bit frustrating, but that seems to be the way that most people use it. Personally I think that the current implementation is great.
@gurgeous can you please just remove the comment from the test? Then I'll approve it :)

Reply:
And resolve conflicts, of course

Reply:
👍